Logic/StandardMessages.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logic
+{
+    public class StandardMessages
+    {
+         public static void StartMessage()
+        {
+            Console.Clear();
+            Console.WriteLine(@" ____  ____   _    ____ _____   ____   _    ____  _  __
+/ ___||  _ \ / \  / ___| ____| |  _ \ / \  |  _ \| |/ /
+\___ \| |_) / _ \| |   |  _|   | |_) / _ \ | |_) | ' / 
+ ___) |  __/ ___ | |___| |___  |  __/ ___ \|  _ <| . \ 
+|____/|_| /_/   \_\____|_____| |_| /_/   \_|_| \_|_|\_\");
+
+            Console.WriteLine("");
+        }
+
```

Comment:
Ni skulle kunna göra en generisk metod som heter "ResponseMessage" eller liknande som tar en sträng i form av felmeddelande in istället för att skapa upp olika metoder för varje scenario, blir lite mer kod i anropen men skalar ner denna klassen avsevärt